doc/source/guide/data_types/spectra.rst

```diff
@@ -97,7 +97,7 @@ To get rid of the noise, we could also clip low intensities by setting vmin.
 
 .. plot::
 
-    from matplotlib import pyplot as plt
+    import matplotlib.pyplot as plt
     import sunpy
     import sunpy.data.sample
     from sunpy.spectra.sources.callisto import CallistoSpectrogram
```

Comment:
well... a few lines below uses `BIR_20110922_101500_01.fit'` and another one, so they have to come from the net... also the `from_range()` calls the files and the extent which I don't see it in here...